php/class-gsuite-provision-settings.php

```diff
@@ -10,24 +10,113 @@ class GSuite_Provision_Settings {
 	 * Adds action hook for menu setup.
 	 */
 	public static function setup() {
-		add_action( 'admin_menu', [ self::$instance, 'admin_menu' ] );
+		if ( self::is_network_enabled() && is_network_admin() ) {
+			add_action( 'network_admin_menu', [ self::$instance, 'network_admin_menu' ] );
+			add_action( 'network_admin_edit_gsuite_provision', [ self::$instance, 'network_admin_menu_process' ] );
+		} else if ( ! self::is_network_enabled() ) {
+			// Do not show the individual site config if the plugin is network-enabled.
```

Comment:
No, this is the intended behavior. If the plugin is network enabled, the settings section should _not_ be shown on individual sites since they'll all use the network-wide credentials to trigger the oauth login.